src/features/Classes/ClassesTable/columns.jsx

```diff
@@ -112,26 +161,32 @@ const columns = [
             alt="menu for actions"
           />
           <Dropdown.Menu>
-            <Dropdown.Item onClick={openModal}>
-              <i className="fa-regular fa-pen-to-square mr-2 mb-1" />
-              Edit Class
-            </Dropdown.Item>
             <Dropdown.Item
+              target="_blank"
+              rel="noreferrer"
               onClick={() => setAssignStaffRole(`${getConfig().LEARNING_MICROFRONTEND_URL}/course/${classId}/home`, classId)}
               className="text-truncate text-decoration-none custom-text-black"
             >
-              <i className="fa-solid fa-arrow-up-right-from-square mr-2 mb-1" />
+              <i className="fa-regular fa-eye mr-2 mb-1" />
               View class content
             </Dropdown.Item>
             <Dropdown.Item>
               <LinkWithQuery
-                to={`/manage-instructors/${encodeURIComponent(masterCourseId)}/${encodeURIComponent(classId)}?previous=classes`}
+                to={`/manage-instructors/${encodeURIComponent(masterCourseId)}/${encodeURIComponent(classId)}`}
                 className="text-truncate text-decoration-none custom-text-black"
               >
                 <i className="fa-regular fa-chalkboard-user mr-2 mb-1" />
                 Manage Instructors
               </LinkWithQuery>
             </Dropdown.Item>
+            <Dropdown.Item onClick={openModal}>
+              <i className="fa-solid fa-pencil mr-2 mb-1" />
+              Edit Class
+            </Dropdown.Item>
+            <Dropdown.Item onClick={handleOpenDeleteModal} className="text-danger">
```

Comment:
Same here, the fetch data is filtered by the course id and in this page should show all classes.

Reply:
This is not solved